### PR TITLE
Added ssl_ciphers

### PR DESCRIPTION
**Pull request by kramvan1**
Allow ssl ciphers to be configured

### CODE REVIEW

**Comment by kramvan1** [comment]
@jjasghar This is another one we would like to get in this week, it's also just basic config options.

**Comment by kramvan1** [comment]
@cmluciano @jjasghar  Humm, seems travis failure is NOT related to this change, but rather to this one: https://github.com/jjasghar/rabbitmq/pull/236   Looks like we pulled out the case to include the yum-epel and yum-erlang, but did not cleanup the specs or the metadata.

**Comment by cmluciano** [comment]
Yes that was on purpose. The tests passes with my changes. The reason that they are failing now is due to the removal from the erlang package recipe change.

**Comment by kramvan1** [comment]
@cmluciano thx for the update.  I have removed those tests from the spec with my latest push.

File: spec/default_spec.rb [hunk 1]
```diff
@@ -56,6 +56,12 @@
       :mode => 00644)
   end
 
+  it 'allows ssl ciphers' do
+    node.set['rabbitmq']['ssl_ciphers'] = ['ecdhe_ecdsa,aes_128_cbc,sha256', 'ecdhe_ecdsa,aes_256_cbc,sha']
+    expect(chef_run).not_to render_file('/etc/rabbitmq/rabbitmq.config').with_content(
```
Comment (michaelklishin) [inline spec/default_spec.rb]: Is it just me or `not_to` should have been `to` here?

Comment (kramvan1) [inline spec/default_spec.rb]: yup, will clean that up and add in the default case of no specific ciphers.

**Pull request merged**